### ISSUE TITLE
Loss function

### ISSUE THREAD

**Issue by eran-shahar** [yluo42/TAC#7]
Hello
Can you please provide instructions on how to use the suggested loss function?
Is the loss used for training contained in the sdr.py file? If so, how to call it? and if not, is there an implementation you can reference to?

**Comment by yluo42** [comment]
By default the model uses: 

https://github.com/yluo42/TAC/blob/e1965f7d85171c4bd3cbc380cb20ddad9981343d/utility/sdr.py#L94

as the training objective. Please refer to the annotation of this function on how to call it.

**Comment by eran-shahar** [comment]
Hi,
Thanks for the quick answer.
When using the above function as the loss functions, the loss starts as negative.
When adding a minus sign to the loss, the loss starts positive and decreases to negative.
What am I doing wrong? what is the right usage?

Another question - how many epochs is usually neccesary?

Thanks and regards

**Comment by yluo42** [comment]
I don't quite understand your question. It's correct to add a minus sign as you have to maximize SI-SDR.

It typically takes more than 70 epochs before early stopping in my experiments.
